design-doc-template.adoc

```diff
@@ -44,36 +44,48 @@ promoted-by:
 :idprefix:
 :idseparator:       -
 
-__<The entire document should be one to two pages long. We will write each analysis document as if it is a conversation
+////
+The entire document should be one to two pages long. We will write each analysis document as if it is a conversation
 with a future developer. This requires a good writing style, with full sentences organized into paragraphs. Bullets are
-acceptable only for visual style, not as an excuse for writing sentence fragments.>__
+acceptable only for visual style, not as an excuse for writing sentence fragments.
+////
 
 == Overview
 
-__<Define the requirement here. Be clear and succinct. You should be able to clearly define the context or problem in two
+////
+Define the requirement here. Be clear and succinct. You should be able to clearly define the context or problem in two
 or three paragraphs (if not sentences). Try to define the problem in the overall context and not to get into too much
-technical detail at this point.>__
+technical detail at this point.
+////
 
 === User Stories
 
-__<Provide one or more brief user stories that illustrate the intended users of the feature and the goal they will seek
-to achieve by using the feature.>__
+////
+Provide one or more brief user stories that illustrate the intended users of the feature and the goal they will seek
+to achieve by using the feature.
+////
 
 == Issue Metadata
 
 === Related Issues
 
-__<List the issues related to this feature>__
+////
+List the issues related to this feature
+////
 
 === Affected Projects or Components
 
-__<List the projects or components that are affected by the feature. List them using their Git repositories.>__
+////
+List the projects or components that are affected by the feature. List them using their Git repositories.
+////
 
```

Comment:
Keep as a visible comment and force the writer to delete so reviewer doesn't have to process it.

In theory the "List them using their Git repositories" could be adoc to driver reviewers to enforce that. I don't know how much we actually care about enforcing it though.